distributions/aws/aws-efs-csi-driver/base/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kube-system
```

Comment:
did you test the notebook and training job in different namespace ?

Reply:
No - I have only done simple testing as mentioned in the description for now.